workflow/kfdrc_consensus_calling.cwl

```diff
@@ -19,6 +19,11 @@ inputs:
   output_basename: string
   annotation_vcf: {type: File, secondaryFiles: ['.tbi'], doc: "VCF of annotions to add to consensus variants, e.g. gnomAD allele frequency"}
   vep_cache: {type: File, doc: "tar gzipped cache from ensembl/local converted cache"}
+  annot_columns: {type: string?, default: 'INFO/AF', doc: "column from annotation_vcf to add to consensus VCF"}
+  filter_names: {type: 'string[]?', default: [ "NORM_DP_LOW", "GNOMAD_AF_HIGH" ], doc: "Names of filters to be added to consensus VCF"}
+  depth_lowerbound: {type: string?, default: '7', doc: "Normal-sample read depth at which to apply depth filter"}
+  frequency_upperbound: {type: string?, default: '0.001', doc: "Population allele frequency above which to apply frequency filter"}
```

Comment:
This should be changed to float, and value should be `0.001` with no quotes. In the context of passing to the tool itself in the workflow, it'll still be interpreted correctly.